DatadogRUM/Sources/Integrations/WebViewEventReceiver.swift

```diff
@@ -100,13 +100,17 @@ internal final class WebViewEventReceiver: FeatureMessageReceiver {
 
                 if var session = event["session"] as? JSON {
                     session["id"] = rum.sessionID
+                    // RUM-4086: unset `has_replay` if native replay is disabled
```

Comment:
Not sure if we need to quote the ticket number in both cases. Probably comment explaining is enough. We can track ticket through the PR